docs/source/whats-new.md

```diff
@@ -4,6 +4,44 @@ What's new
 See [GitHub releases page](https://github.com/OSOceanAcoustics/echopype/releases) for the complete history.
 
 
+# v0.7.2 (2023 July 14)
```

Comment:
```suggestion
# v0.8.0 (2023 July 14)
```

Should probably also change the date from `July 14`

Reply:
Yes! it depends on #1056 and #1057